babel/numbers.py

```diff
@@ -717,10 +722,16 @@ def parse_decimal(string, locale=LC_NUMERIC, strict=False):
                 ), suggestions=[proper])
             else:
                 proper_alt = format_decimal(parsed_alt, locale=locale, decimal_quantization=False)
-                raise NumberFormatError((
-                    "%r is not a properly formatted decimal number. Did you mean %r? Or maybe %r?" %
-                    (string, proper, proper_alt)
-                ), suggestions=[proper, proper_alt])
+                if proper_alt == '0':
```

Comment:
Hmm, could we just look at whether `proper_alt != proper`? That'd be bullet-proof...

Reply:
You're right, that's a way better check (well, doing `proper_alt == proper` as not to change the order of NumberFormatError's and make the diff larger).